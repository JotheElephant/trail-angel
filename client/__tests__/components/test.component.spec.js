import React from 'react';
import renderer from 'react-test-renderer';

import TestComponent from '../../src/components/test.component';


it('renders a test component view using Snapshots', () => {
  const testIncrement = jest.fn();

  const tree = renderer.create(
    <TestComponent
      testIncrement={testIncrement}
      testDecrement={testIncrement} />
  ).toJSON();

  expect(tree).toMatchSnapshot();

});