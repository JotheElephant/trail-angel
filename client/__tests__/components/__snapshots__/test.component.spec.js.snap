exports[`test renders a test component view using Snapshots 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "flex": 1,
      "justifyContent": "center",
    }
  }>
  <View
    accessibilityComponentType={undefined}
    accessibilityLabel={undefined}
    accessibilityTraits={undefined}
    accessible={true}
    hitSlop={undefined}
    onLayout={undefined}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "lightgreen",
        "height": 30,
        "justifyContent": "center",
        "margin": 3,
        "opacity": 1,
        "padding": 10,
        "width": 100,
      }
    }
    testID={undefined}>
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail">
      more
    </Text>
  </View>
  <View
    accessibilityComponentType={undefined}
    accessibilityLabel={undefined}
    accessibilityTraits={undefined}
    accessible={true}
    hitSlop={undefined}
    onLayout={undefined}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "red",
        "height": 30,
        "justifyContent": "center",
        "margin": 3,
        "opacity": 1,
        "padding": 10,
        "width": 100,
      }
    }
    testID={undefined}>
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail">
      less
    </Text>
  </View>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail" />
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail" />
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail" />
</View>
`;
